devicetypes/smartthings/viconics-schneider-room-controller.src/viconics-schneider-room-controller.groovy

```diff
@@ -0,0 +1,550 @@
+/**
+ *  Viconics/Schneider Room Controller
+ *
+ *  Copyright 2020
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ * 	  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+import groovy.json.JsonOutput
+import physicalgraph.zigbee.zcl.DataType
+metadata {
+	definition(name: "Viconics Schneider Room Controller", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.thermostat", mcdSync: true) {
+
+		capability "Actuator"
+		capability "Sensor"
+		capability "Occupancy Sensor"
+		capability "Temperature Measurement"
+		capability "Relative Humidity Measurement"
+		capability "Thermostat"
```

Comment:
I think you should just leave this deprecated capability off

Reply:
I'm not sure about it. When I got rid of that line, all  other plugin elements that are related to the Thermostat weren't displayed.

Reply:
some metadata is defined by referencing the `Thermostat` attributes, but others, such as Z-Wave Battery Thermostat, specifically reference the discrete capabilities. You might just need to poke at the metadata bit.

Reply:
@greens i will give it a try ( merge request with UI metadata wasn't created yet)

Reply:
Yeah, if the metadata references `/capability/thermostat/X` where `X` is some number then it is referring to the `Thermostat` capability. If you drop this and use metadata that references the individual capabilities it should work.

Reply:
Thank You @dkirker, I removed those references from UI metadata and it works now.